test/libdnf/preserve_order_map/test_preserve_order_map.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "test_preserve_order_map.hpp"
+
+#include "libdnf/common/preserve_order_map.hpp"
+
+#include <string>
+#include <vector>
+
+
+CPPUNIT_TEST_SUITE_REGISTRATION(PreserveOrderMapTest);
+
+
+void PreserveOrderMapTest::setUp() {}
+
+
+void PreserveOrderMapTest::tearDown() {}
```

Comment:
You don't need to override these if they're empty?

Reply:
It is true. I removed it.